Apps/Admin/Server/Services/InactiveUserService.cs

```diff
@@ -100,7 +102,10 @@ public async Task<RequestResult<List<AdminUserProfileView>>> GetInactiveUsers(in
         // Compare inactive users in DB to users in Keycloak
         if (inactiveProfileResult.Status == DbStatusCode.Read && activeProfileResult.Status == DbStatusCode.Read)
         {
-            inactiveUsers.AddRange(this.autoMapper.Map<IEnumerable<AdminUserProfile>, IList<AdminUserProfileView>>(inactiveProfileResult.Payload));
+            inactiveUsers.AddRange(
+                inactiveProfileResult.Payload.Select(
+                        x => AdminUserProfileMapUtils.ToUiModel(x, this.configuration, true, this.autoMapper))
```

Comment:
now the configuration must be injected here, another dependency to manage, I personally don't like this.

Reply:
Agreed. Changed.